build.cake

```diff
@@ -0,0 +1,47 @@
+var target = Argument("target", "Default");
+
+Task("Restore")
+  .Does(() =>
+{
+    DotNetCoreRestore("src/\" \"tests/");
+});
+
+Task("Build")
+    .IsDependentOn("Restore")
+  .Does(() =>
+{
+    DotNetCoreBuild("src/**/project.json\" \"tests/**/project.json");
```

Comment:
Won't build on *nix (travis) - there is no full framework there. You need something like:
```
	var projects = GetFiles("./src/**/project.json");
	foreach(var projectFile in projects)
	{
		var frameworks = ParseJsonFromFile(projectFile)
			.GetValue("frameworks")
			.Children<JProperty>()
			.Select(j => j.Name);

		foreach(var framework in frameworks)
		{
			if (!IsRunningOnWindows() && framework == "net452")
				continue;
			
			Information("Framework version {0}", framework);
			var settings = new DotNetCoreBuildSettings
			{
				Framework = framework
			};
			DotNetCoreBuild(projectFile.FullPath, settings);
		}			
	}
```